easybuild/easyconfigs/r/R/R-3.1.1-intel-2014b.eb

```diff
@@ -222,6 +222,7 @@ exts_list = [
     ('GenomicAlignments', '1.0.4', bioconductor_options),
     ('ShortRead', '1.22.0', bioconductor_options),
     ('graph', '1.42.0', bioconductor_options),
+    ('doMC', '1.3.3', bioconductor_options),
```

Comment:
xcms does depend on doMC, so can't just flesh this out, but I Can change the entire pull request to target a newer R...